sympy/geometry/curve.py

```diff
@@ -145,6 +145,26 @@ def arbitrary_point(self, parameter='t'):
                 'and cannot be used as a parameter.' % tnew.name)
         return Point(*[w.subs(t, tnew) for w in self.functions])
 
+    def parametric_region(self):
+        """
+        Returns an object of ParametricRegion class representing the Curve.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import t
+        >>> from sympy import Curve
+        >>> c = Curve((t**3, 4*t), (t, -3, 4))
+        >>> c.parametric_region()
+        ParametricRegion((t**3, 4*t), (t, -3, 4))
+
+        """
+        from sympy.vector import ParametricRegion
```

Comment:
I would try to avoid making the geometry module dependent on the vector module. Have you thought about using `functools.singledispatch` in the vector module instead?

Reply:
Thanks. I was not familiar with it. I will try using it.